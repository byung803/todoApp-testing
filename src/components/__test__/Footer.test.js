import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../Footer';

it('should have same snapshot Footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
})