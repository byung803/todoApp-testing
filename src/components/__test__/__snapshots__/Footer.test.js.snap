// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have same snapshot Footer 1`] = `
<div>
  <Connect(Link)
    visibilityFilter="SHOW_ALL"
  >
    Show All
  </Connect(Link)>
  <Connect(Link)
    visibilityFilter="SHOW_ACTIVE"
  >
    Active
  </Connect(Link)>
  <Connect(Link)
    visibilityFilter="SHOW_COMPLETED"
  >
    Completed
  </Connect(Link)>
</div>
`;
